### PR TITLE
Add orchestration cluster glossary terms

### PR DESCRIPTION
**Pull request by akeller**
## Description

~This is a DRAFT because a crisp definition is not found internally. This PR will need to be nurtured by an @camunda/tech-writers to work with @felix-mueller(?) on a definition.~

We should also mention the 1:many relationship between ~a management cluster~ Web Modeler + Console and an orchestration cluster in SM, but I wasn't sure if we wanted to do that here when the definition should be crisp with a link to more info. 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/reference/glossary.md [hunk 1]
```diff
@@ -149,6 +149,10 @@ The log is comprised of an ordered sequence of records written to persistent sto
 
 - [Partitions](/components/zeebe/technical-concepts/partitions.md#partition-data-layout)
 
+### Management cluster
```
Comment (theburi) [inline docs/reference/glossary.md:152]: Lets call it "Web Modeler and Console clusters" we are not planning to use Management cluster for now.

File: docs/reference/glossary.md [hunk 2]
```diff
@@ -163,6 +167,10 @@ A message contains information to be delivered to interested parties during exec
 
 - [Messages](/components/concepts/messages.md)
 
+### Orchestration cluster
+
+An orchestration cluster includes Zeebe, Operate, Tasklist, and Optimize, existing alongside the management cluster.
```
Comment (theburi) [inline docs/reference/glossary.md:172]: Orchestration cluster= Orchestration Cluster Core + Connectors+ Connectors Runtime
 Orchestration Cluster Core = Zeebe, Operate, Tasklist,  Optimize and identity

**Comment by akeller** [comment]
@theburi I followed up on your [doc](https://docs.google.com/document/d/1Bb8Yx8a4BN7Q4byGKN33IB_eggS-Av9j-u4mhU27oeQ/edit?tab=t.0) since it sounds like some of this has changed while I was out. See my comments, particularly regarding SaaS and "core".

File: docs/reference/glossary.md (hunk 1 above)
Comment (akeller) [reply docs/reference/glossary.md]: I removed this term

File: docs/reference/glossary.md (hunk 2 above)
Comment (akeller) [reply docs/reference/glossary.md]: I added both terms, but referred to the "Orchestration Cluster Core" as follows:

> ### Orchestration core
> An orchestration core or orchestration cluster core includes Zeebe, Operate, Tasklist, Optimize, and Identity.

Would this work @theburi? Or do you want use to be specific and refer to this as "orchestration cluster core"?

**Comment by akeller** [comment]
@theburi can you review these terms for good working definitions? We can refine them in another PR, but I want to make sure the tech writers know how to use these moving forward.

**Review by conceptualshark — COMMENTED** [review]
This matches my understanding of orchestration cluster vs core, with two comments:

- As far as I understand, the Core will not change, but additional features might be added to the orchestration cluster - IDP and RPA? So the definition as written will need to be expanded when that occurs. I'm wondering if we have a more generic way to refer to "The orchestration core and any other optional components"? 
- From what I've seen in discussions and PRs, we're using "orchestration cluster" over "core" almost exclusively. Is "core" something we want to push externally? Does it matter to a customer if they know "core" is only X components, or can they just assume their orchestration cluster is _always_ X components, and also inclusive of anything else added to it? If we rarely use "core" in the docs it can make the communication a little more confusing. More of a question for @theburi.

**Review by conceptualshark — APPROVED** [review]
Based on further discussion with @theburi, going ahead and approving this, as it matches what was confirmed as approved in [this internal doc](https://docs.google.com/document/d/1Bb8Yx8a4BN7Q4byGKN33IB_eggS-Av9j-u4mhU27oeQ/edit?tab=t.0#heading=h.cwo8qxehm1w4).

We'll just need to remember that if/when additional features get added to the orchestration cluster, they will need to be added to this list.

**Pull request merged**